lib/dat-manager.js

```diff
@@ -19,18 +18,22 @@ module.exports = class Manager extends EventEmitter {
     })
   }
   _watch (dat) {
-    dat.on('update', () => this.emit('update'))
+    dat.events.on('update', () => this.emit('update'))
     return dat
   }
   get () {
     return this.dats
   }
   create (dir, cb) {
-    const dat = Dat({ dir })
-    dat.owner = true
-    dat.dir = dir
-    dat.share(err => {
+    Dat(dir, (err, dat) => {
       if (err) return cb(err)
+
+      dat.dir = dir
```

Comment:
Oops sorry changed API there. I can expose this if that'd be helpful. `dat.path` is the full resolved path but `dat.dir` could be the relative `dir` passed in.